sources/packages/web/src/views/aest/student/SINManagement.vue

```diff
@@ -198,7 +198,7 @@ export default {
         processingEditExpiryDate.value = true;
         const formioForm =
           modalResult as FormIOForm<UpdateSINValidationAPIInDTO>;
-        formioForm.data.expiryDate = getISODateOnlyString(
+        formioForm.data.expiryDate = dateOnlyLongString(
```

Comment:
Please revert this change. This is formatting the date to be sent to the API, not to the UI.

Reply:
Noted